pkg/operator/ceph/cluster/mgr/config.go

```diff
@@ -87,10 +88,20 @@ func (c *Cluster) generateKeyring(m *mgrConfig) (string, error) {
 		return "", err
 	}
 
+	if c.shouldRotateCephxKeys {
+		logger.Infof("rotating cephx key for mgr daemon %q in the namespace %q", m.ResourceName, c.clusterInfo.Namespace)
+		newKey, err := s.RotateKey(user)
+		if err != nil {
+			return "", errors.Wrapf(err, "failed to rotate cephx key for mgr daemon %q", m.ResourceName)
+		} else {
```

Comment:
As we are returning inside the if statment, we really dont need else here isnt it?

Reply:
done.